.circleci/config.yml

```diff
@@ -142,7 +141,8 @@ jobs:
           name: Run PHPUnit tests
           command: |-
             mkdir -p build/logs/tmp build/cov
-            phpdbg -qrr vendor/bin/phpunit --coverage-php build/cov/coverage.cov --log-junit build/logs/tmp/coverage.xml --colors=always
+            circleci tests glob "tests/**/*Test.php" | circleci tests split --split-by=timings | parallel -j10% --rpl '{_} s/\//_/g;' --env SYMFONY_DEPRECATIONS_HELPER \
```

Comment:
@dunglas This also proves that #2210 was wrong. `SYMFONY_DEPRECATIONS_HELPER` cannot be read from `phpunit.xml.dist`